Mathlib/Analysis/InnerProductSpace/Basic.lean

```diff
@@ -905,6 +905,17 @@ protected theorem Complex.inner (w z : ℂ) : ⟪w, z⟫_ℝ = (conj w * z).re :
 
 end RCLikeToReal
 
+/-- An `RCLike` field is a real inner product space. -/
+noncomputable instance RCLike.toInnerProductSpaceReal : InnerProductSpace ℝ 𝕜 where
```

Comment:
Could you add the two `example`s I commented with earlier that prove this creates no diamonds?